scripts/update-ois-version.ts

```diff
@@ -0,0 +1,55 @@
+import { readFileSync, writeFileSync } from 'fs';
+import fg from 'fast-glob';
+import { format } from 'prettier';
+
+if (process.argv.length < 3) {
```

Comment:
Maybe it could parse the version from package.json?

Reply:
You mean add a new field for OIS version within package.json and pull from there?

Reply:
I meant that OIS is already a dependency in the Airnode packages, so we might use that (since the package version dictates the OIS version).

Reply:
I think we have different ideas for the flow of bumping the OIS version. The way the script is implemented currently:
1. New OIS version gets released e.g. `2.1.0` from the ois repo
2. Run this script in the Airnode repo: `yarn update-ois-version 2.1.0`. This updates the 3 `package.json` files that OIS appears in, as well as all `.json` and `.ts` files where `oisFormat appears`.
3. `yarn run bootstrap && yarn run build`

What it sounds like you're suggesting:
1. Same as above
2. Bump ois package using a conventional approach (building / updating `yarn.lock` happens within here)
3. Run this script minus needing to enter the version and instead pull the version from a `package.json` where ois is a dependency e.g. `airnode-validator/package.json`

I can see it either way.

Reply:
Perhaps @amarthadan you had a flow in mind when creating the issue?

Reply:
I would probably go with the second one. My reasoning is that the renovate bot will open a PR for the ois package version bump anyway. In the second flow, we just run the script on that branch and update the PR.
If we would go with the first flow I suspect we would end up doing pretty much the same thing because the person releasing ois package is not usually the same bumping Airnode dependencies to do it immediately after :smile:

Reply:
Sounds good, I'll revise.